src/app/pages/OnDemandRadioPage/index.test.jsx

```diff
@@ -20,6 +20,13 @@ import withMediaError from '#lib/utilities/episodeAvailability/withMediaError';
 
 const OnDemandRadioPage = withMediaError(_OnDemandRadioPage);
 
+const toggles = {
```

Comment:
Can we possibly add a test that recreates the live issue?  eg, a test where `toggles` is missing data, is null, or nonsense value?